### PR TITLE
Port QWT to Qt5 for brew

### PR DESCRIPTION
**Pull request by j-rivero**
Only for reviewing proposes. @scpeters could you please give it a look?

Do I need to build bottles and provide hashes for updating the formula? If so, any advice about how to do it?

### CODE REVIEW

**Comment by scpeters** [comment]
Thanks, I'll take a look

File: Formula/qwt.rb [hunk 1]
```diff
@@ -4,18 +4,10 @@ class Qwt < Formula
   url "https://downloads.sourceforge.net/project/qwt/qwt/6.1.3/qwt-6.1.3.tar.bz2"
   sha256 "f3ecd34e72a9a2b08422fb6c8e909ca76f4ce5fa77acad7a2883b701f4309733"
 
-  bottle do
```
Comment (j-rivero) [inline Formula/qwt.rb:7]: [Looks like we don't need to provide hashes](https://github.com/ogrisel/homebrew/blob/master/Library/Contributions/example-formula.rb#L94) when submitting changes to homebrew.

**Comment by scpeters** [comment]
I'm building it now

**Comment by scpeters** [comment]
It builds for me just fine, though there are still run-time seg-faults with qt5 :(

**Comment by j-rivero** [comment]
oh. How can I give it a try in runtime?

**Comment by scpeters** [comment]
sorry, I meant that I built gazebo against this formula and qt5, and that gazebo fails in runtime

**Comment by j-rivero** [comment]
I've created the PR against the homebrew-core repo. Let's see.

**Comment by scpeters** [comment]
I suspect they are waiting for something specific, but we'll see what they say.

**Comment by scpeters** [comment]
https://github.com/Homebrew/homebrew-core/pull/5162

**Pull request closed without merging**